compiler+runtime/include/cpp/jank/jit/processor.hpp

```diff
@@ -34,11 +35,17 @@ namespace jank::jit
     void load_bitcode(native_persistent_string const &module,
                       native_persistent_string_view const &bitcode) const;
 
+    string_result<void> remove_symbol(native_persistent_string const &name) const;
+
     template <typename T>
-    T find_symbol(native_persistent_string const &name) const
+    string_result<T> find_symbol(native_persistent_string const &name) const
     {
-      auto const sym(interpreter->getSymbolAddress(name.c_str()).get());
-      return sym.toPtr<T>();
+      if(auto symbol{ interpreter->getSymbolAddress(name.c_str()) })
+      {
+        return symbol.get().toPtr<T>();
+      }
+
+      return err("Failed to find symbol");
```

Comment:
Please use `Failed to find symbol '{}'` and format in the symbol. We can all be helping with error reporting by generating better errors. :)